docs/content/en/docs-v0.18.x/user-guide/command-line-tool.md

```diff
@@ -16,11 +16,11 @@ You can use pipectl to add and sync applications, wait for a deployment status.
 1. Download the appropriate version for your platform from [PipeCD Releases](https://github.com/pipe-cd/pipe/releases).
 
     We recommend using the latest version of pipectl to avoid unforeseen issues.
-    Please set `{VERSION}` to the same format like `v0.9.15`.
-    And `{OS}` can be replaced with either `linux` or `darwin`.
+    Run the following script:
 
     ``` console
-    curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/{VERSION}/pipectl_{VERSION}_{OS}_amd64
+    OS="darwin" # or "linux"
+    curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/{{< blocks/latest_version >}}/pipectl_{{< blocks/latest_version >}}_${OS}_amd64
```

Comment:
```suggestion
    curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/v0.18.0/pipectl_v0.18.0_${OS}_amd64
```
nits, since this docs is for version `v0.18.x`, but I'm not sure should we keep it version specified or just pass the latest version name value here 😅

Reply:
fmm... I see.
Do you need a variable like `{{< blocks/build_version >}}` for the each version of document?
Otherwise, maintenance costs will be high. 🤔

Reply:
> Do you need a variable like {{< blocks/build_version >}} for the each version of document?

Nice idea 👍  but tbh, I don't know how do we do to get the value for that variable. Do you have any idea for that?
The current `{{< blocks/latest_version >}}` variable defined here
https://github.com/pipe-cd/pipe/blob/master/docs/layouts/shortcodes/blocks/latest_version.html
Basically, we use the `RELEASE` environment variable in the docs build instance as the value.

Reply:
That would be perfect but currently, we don't have a nice way to fetch that version so let me merge this PR for now.